src/server/index.js

```diff
@@ -76,11 +77,16 @@ Promise.all([fetchParameters(), fetchVariables()])
       assoc('itemType', 'variable'),
       addNormalizedDescription(variablesResponse.data),
     )
+
+    console.log("Fetching text translations...")
```

Comment:
Removed but there are 10 older occurrences of `console.log` in the same file. In the future, what should we use instead ?